src/components/CookieBanner/CookieBanner.test.js

```diff
@@ -8,22 +8,21 @@ Enzyme.configure({adapter: new Adapter17()});
 describe('feature', () => {
 
   it('should show cookie banner', () => {
-    //const {} = mockCallBack;
     const wrapper = mount(<CookieBanner></CookieBanner>);
     wrapper.find('.util-cookie-banner__cookie-icon-show').simulate('click');
     expect(wrapper.exists('.util-cookie-banner-container--show')).toEqual(true);
   })
 
   it('should hide cookie banner', () => {
-    //const {} = mockCallBack;
     const wrapper = mount(<CookieBanner><CgClose></CgClose></CookieBanner>);
```

Comment:
Do we need to mount the Icon in the test?

Reply:
I have removed the icon from the mount.